test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,21 @@
+require "test_helper"
+
+describe VotesController do
+ it "will add user to vote" do 
```

Comment:
You should also try voting for the same work twice and voting without being logged in.